packages/core/src/accessibility/focusStyleManager.ts

```diff
@@ -4,12 +4,20 @@
  * of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
  * and https://github.com/palantir/blueprint/blob/master/PATENTS
  */
+/* istanbul ignore next */
```

Comment:
How does one try it locally? I can't find a task, and coverage doesn't run on `gulp test` for me the way it does on CircleCI